LayoutTests/fast/forms/color/color-input-swatch-expected.txt

```diff
@@ -1,102 +1,85 @@
 initial state
 
 getComputedStyle(swatch).backgroundColor is rgb(0, 0, 0)
-getComputedStyle(swatch).backgroundImage is none
-getComputedStyle(swatch).backgroundSize is auto
+swatch.innerHTML is
```

Comment:
This doesn't look quite right. Maybe you want a `shouldBe`-like function here to clearly state that empty string is the expectation? Other logged values may benefit from `shouldBe`, too.

Reply:
I don't really want to state the expectations as they can differ per platform, as they do here.

Reply:
Even this one, empty `innerHTML`?

That said, color components differing per platform (and I'm sure also per configuration) is only more of a reason to have expectations. Possibly, saying that the numbers should be within a certain range. Currently, you are asking everyone who sees the test fail to investigate for themselves whether the different number means failing or passing, and that's rather hard.

Reply:
They only differ between macOS and everywhere else at this point. And only macOS has a deeper subtree. Anyone touching the shadow tree for this component would know why this test regressed.

Reply:
I hear your explanations, but I still really dislike tests that just dump results, as existing ones that do so are a constant nuisance.